File: leetcode/editor/en/[215]Kth Largest Element in an Array.js
//leetcode submit region begin(Prohibit modification and deletion)
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var findKthLargest = function (nums, k) {
    nums.sort((a, b) => a - b);
    return nums[nums.length - k];
};
//leetcode submit region end(Prohibit modification and deletion)

File: editor/en/[215]Kth Largest Element in an Array.java
215
        Kth Largest Element in an Array
        2022-12-09 14:55:35

//leetcode submit region begin(Prohibit modification and deletion)
class Solution {
    public int findKthLargest(int[] nums, int k) {
        Arrays.sort(nums);
        return nums[nums.length - k];
    }
}
//leetcode submit region end(Prohibit modification and deletion)
